services/121-service/src/payments/fsp-integration/intersolve-visa/entities/intersolve-visa-wallet.entity.ts

```diff
@@ -4,7 +4,8 @@ import { Base121Entity } from '@121-service/src/base.entity';
 import { IntersolveVisaCustomerEntity } from '@121-service/src/payments/fsp-integration/intersolve-visa/entities/intersolve-visa-customer.entity';
 import { maximumAmountOfSpentCentPerMonth } from '@121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.const';
 
-enum IntersolveVisaWalletStatus {
+// Export this to use in migrate-visa.service
```

Comment:
Then it should be in its own file. Or if this is "temporary" then add a TODO comment to make it private again after we remove the migration code from the code base.

=> Should we have an item on the PB to "clean up the code base" after the migration is done?

Reply:
This file will be removed after the migration. But yes makes sense to create a PB for that.

Reply:
Ok, I added a link to this convo there.